_hacks/17.md

```diff
@@ -1,5 +1,15 @@
 ---
 hack_number: 17
 date: 2021-08-28
-happened: false
+happened: true
 ---
+
```

Comment:
Maybe `it was a bit difficult to get people interested in a remote hack` -> `we had a quieter remote hack`?